numpy/random/mtrand/mtrand.pyx

```diff
@@ -573,283 +573,6 @@ def _shape_from_size(size, d):
            shape = tuple(size) + (d,)
     return shape
 
-
```

Comment:
Rebased this PR onto #8088 to double check that the rebasing wouldn't explode.  Will rebase once more as soon as #8088 is merged.